newsfragments/314.bugfix

```diff
@@ -0,0 +1 @@
+CBF (de)compression: fix bugs in image compression and decompression
```

Comment:
Could you expand on under what circumstances this bug is applicable? As far as I am aware it was only (noticed) in `dlsnxs2cbf` - could it affect reading "normal" cbfs?

Reply:
More news added. I don't know yet if this could affect reading of "normal" CBF images, I will re-review in a moment.

Reply:
Re-reviewed